_includes/mermaid.html

```diff
@@ -0,0 +1,22 @@
+<script type="text/javascript"
+  src="https://unpkg.com/mermaid@8.10.2/dist/mermaid.min.js">
+</script>
+<script type="text/javascript" 
+    src="https://unpkg.com/jquery@3.6.0/dist/jquery.min.js">
+</script>
+
+<script>
+jQuery(document).ready(function() {
```

Comment:
Could we drop jQuery completely?

```suggestion
document.addEventListener('DOMContentLoaded', function() {
```